validation/jvm/src/main/scala/hmda/validation/rules/ts/syntactical/S028.scala

```diff
@@ -0,0 +1,15 @@
+package hmda.validation.rules.ts.syntactical
+
+import hmda.model.fi.ts.TransmittalSheet
+import hmda.validation.dsl.{ HmdaDsl, CommonDsl, Result }
+
+/*
+ Timestamp must be numeric and in ccyymmddhhmm format
+ */
+object S028 extends CommonDsl with HmdaDsl {
+  def apply(ts: TransmittalSheet): Result = {
+    import scala.language.postfixOps
+    val timestamp = ts.timestamp
+    (timestamp is numeric) and (timestamp.toString is validTimestampFormat)
```

Comment:
@schbetsy and I noticed that `ts.timestamp` is always numeric, because it is a Long. As the parser currently stands, a TS string with a non-numeric timestamp would fail during parsing. Will the parser eventually be responsible for checks such as S028, or is this a sign that `TransmittalSheet.timestamp` should be a String, so that this check can happen during validation?